src/axom/spin/tests/spin_bvh.cpp

```diff
@@ -1306,6 +1306,162 @@ void check_find_points_zip2d()
   axom::setDefaultAllocator(current_allocator);
 }
 
+//---------------------------------------------------------------------------
+
+/*!
+ * \brief Makes an array of 2D points for testing the BVH.
+ *
+ * \return points The array of points to use for testing.
+ */
+template <typename FloatType>
+axom::Array<axom::primal::Point<FloatType, 2>> make_query_points_2d()
+{
+  int dims[] = {101, 101};
+  FloatType x0 = -1.5, x1 = 1.5;
+  FloatType y0 = -1.5, y1 = 1.5;
+  axom::Array<FloatType> x(dims[0]), y(dims[1]);
+  axom::numerics::linspace(x0, x1, x.data(), dims[0]);
+  axom::numerics::linspace(y0, y1, y.data(), dims[1]);
+  axom::Array<axom::primal::Point<FloatType, 2>> points;
+  points.reserve(dims[0] * dims[1]);
+  for(int j = 0; j < dims[1]; j++)
+  {
+    for(int i = 0; i < dims[0]; i++)
+    {
+      points.push_back(axom::primal::Point<FloatType, 2>({x[i], y[j]}));
+    }
+  }
+  return points;
+}
+
+//---------------------------------------------------------------------------
+
+/*!
+ * \brief Tests inserting a single point into the BVH and querying a set of
+ *        points against it, using a pattern seen in DistributedClosestPoint.
+ *        The test demonstrates current_node that the BVH returns an in range
+ *        value to the checkMinDist lambda.
+ *
+ * \param [in] bvh The BVH object to use.
+ * \param [in] points The points that will be inserted in the BVH as bboxes.
+ * \param [in] query_pts The array of query points to use against the BVH.
+ * \param [in] none Whether to add no points to the BVH. If this is the case,
+ *                  none of the queries against the BVH will set the minElem
+ *                  value and we use none so we know the expected result.
+ */
+template <typename BVHType, typename PointType>
+void bvh_compute_point_distances_2d(BVHType& bvh,
+                                    axom::Array<PointType>& points,
+                                    axom::Array<PointType>& query_pts,
+                                    bool none)
+{
+  EXPECT_TRUE(!query_pts.empty());
+
+  using ExecSpace = typename BVHType::ExecSpaceType;
+  using FloatType = typename PointType::CoordType;
+  using BoxType = axom::primal::BoundingBox<FloatType, 2>;
+  constexpr int INVALID_ELEMENT_INDEX = -1;
+  constexpr int FIRST_ELEMENT_INDEX = 0;
+
+  // Borrowed from DistibutedClosestPoint.
+  struct MinCandidate
+  {
+    /// Squared distance to query point
+    double minSqDist {numerics::floating_point_limits<double>::max()};
+    /// Index within mesh of closest element
+    int minElem {INVALID_ELEMENT_INDEX};
+  };
+
+  // Initialize the BVH with the points.
+  axom::IndexType npts = none ? 0 : points.size();
+  BoxType* bboxes =
+    axom::allocate<BoxType>(npts ? npts : 1,  // do not allocate 0 elements
+                            axom::execution_space<ExecSpace>::allocatorID());
+  for(axom::IndexType i = 0; i < npts; i++) bboxes[i] = BoxType(points[i]);
```

Comment:
Please use braces on single line if/for's.

https://axom.readthedocs.io/en/develop/docs/sphinx/coding_guide/sec09_format.html#conditional-statements-and-loops